src/Rector/Contrib/PHPUnit/SpecificMethodCountRector.php

```diff
@@ -60,15 +74,25 @@ public function isCandidate(Node $node): bool
             return false;
         }
 
-        return $secondArgumentValue->name->toString() === 'count';
+        $coutableMethod = $secondArgumentValue->name->toString();
+
+        return $coutableMethod === 'count' || $coutableMethod === 'sizeof';
     }
 
     /**
      * @param MethodCall $methodCallNode
      */
     public function refactor(Node $methodCallNode): ?Node
     {
-        $methodCallNode->name = new Identifier('assertCount');
+        $oldMethodName = $methodCallNode->name->toString();
+
+        if ($oldMethodName === 'assertEquals' || $oldMethodName === 'assertSame') {
+            /** @var string $trueMethodName */
+            $methodCallNode->name = new Identifier('assertCount');
+        } elseif ($oldMethodName === 'assertNotEquals' || $oldMethodName === 'assertNotSame') {
+            /** @var string $falseMethodName */
+            $methodCallNode->name = new Identifier('assertNotCount');
+        }
```

Comment:
I'm thinking, there should be some service that handle services rename. It's quite often the case.

So this hard to read code could be replaced by:

```php
$renameMap = [
    'assertEquals' => 'assertCount',
    'assertSame' => 'assertCount'
    'assertNotEquals' => 'assertNotCount',
    'assertNotSame' => 'assertNotCount',
];

$this->methodRenamer->renameNode($node, $renameMap);
```


What do you think? :wink:

Reply:
Good Idea, it also makes easier to add more assertions in the future.

Reply:
Allright, do you want to try it out or should I handle it?

Since we have `Rector\NodeAnalyzer` for analyzing (read-only),
the place for this one could be `Rector\NodeChanger` for changin (write). This one would be  `Rector\NodeChanger\MethodNameChanger` service

Reply:
I'm trying. I let you know if everything blow up :fire:

Reply:
Sorry, now I understood this comment. We should delegate this function to a class, right? `SOLID` :smile: